src/integration/java/org/springframework/data/couchbase/config/CouchbaseEnvironmentNoShutdownProxyIT.java

```diff
@@ -14,7 +14,7 @@
  */
 @RunWith(ContainerResourceRunner.class)
 @ContextConfiguration(classes = IntegrationTestNoShutdownApplicationConfig.class)
-public class CouchbaseEnvironmentNoShutdownProxyTest {
+public class CouchbaseEnvironmentNoShutdownProxyIT {
```

Comment:
Going with the Spring convention `Tests`/`IntegrationTests` prevents the break in naming `Test` vs `IT`.

Reply:
Updated the PR so integration test class names end in `IntegrationTests` instead of `IT`.

Fixed.